sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -288,14 +288,99 @@ public virtual async Task<Response<AccountProperties>> GetAccountPropertiesAsync
 
         #endregion
 
+        #region Copy
+        /// <summary>
+        /// Copy a custom model stored in this resource (the source) to the user specified
+        /// target Form Recognizer resource.
+        /// </summary>
+        /// <param name="modelId">Model identifier of the model to copy to the target Form Recognizer resource.</param>
+        /// <param name="target">A <see cref="CopyAuthorization"/> with the copy authorization to the target Form Recognizer resource.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A <see cref="CopyModelOperation"/> to wait on this long-running operation.  Its <see cref="CopyModelOperation"/>.Value upon successful
+        /// completion will contain meta-data about the model copied.</returns>
+        [ForwardsClientCalls]
+        public virtual CopyModelOperation StartCopyModel(string modelId, CopyAuthorization target, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(modelId, nameof(modelId));
```

Comment:
@kristapratico should we also add this validation for python? or should this validation be done by the service

Reply:
done by service for python

Reply:
we validate that there is something there and we validate it is a well formed UUID

Reply:
I thought that was what we agreed on yesterday, when leaving modelId as string. no?

Reply:
Oh no, I missed that we agreed on that. Just curious - what was rationale that this param should be validated and not others?

Reply:
we validate that all parameters that are required in each method (req parameters from the service) are not null. In this case I validate that `modelId` and `target`.